tsdb/head_test.go

```diff
@@ -173,7 +173,9 @@ func BenchmarkLoadWAL(b *testing.B) {
 				b.ResetTimer()
 
 				// Load the WAL.
-				h.Init(0)
+				for i := 0; i < b.N; i++ {
+					h.Init(0)
```

Comment:
Samples would be added to head only in the first iteration and the rest will be discarded. Should we have creation of a new head inside the loop?

Reply:
:+1: Good call, done.

Reply:
What were the benchmark results after this change?

Reply:
The results in the description were done after this change.